stylesheet.css

```diff
@@ -0,0 +1,104 @@
+header{
+    display: flex;
+    padding: 10px 10px 10px 10px;
+    justify-content: space-around;
+    align-items: center;
+    border: 2px solid lightgray;
+    background-color: white;
+    border-radius: 5px;
+    margin-bottom: 10px;
+    margin-left: 0px;
+    margin-right: 0px; 
+}
+ul li{
+    text-decoration: none;
+    display: inline-block;
+    padding: 20px;
+
+}
+.button1{
```

Comment:
Please give this a more meaningful name instead of button1, name this according to its functionality

Reply:
resolved!